definitions/npm/semver_v7.x.x/flow_v0.104.x-/semver_v7.x.x.js

```diff
@@ -0,0 +1,836 @@
+  import type semverPatch from "semver/functions/patch"
+  import type semverPrerelease from "semver/functions/prerelease"
+  import type semverCompare from "semver/functions/compare"
+  import type semverRcompare from "semver/functions/rcompare"
+  import type semverCompareLoose from "semver/functions/compare-loose"
+  import type semverCompareBuild from "semver/functions/compare-build"
+  import type semverSort from "semver/functions/sort"
+  import type semverRsort from "semver/functions/rsort"
+  declare export {
+    semverParse as parse,
+    semverValid as valid,
+    semverClean as clean,
+    semverInc as inc,
+    semverDiff as diff,
+    semverMajor as major,
+    semverMinor as minor,
+    semverPatch as patch,
+    semverPrerelease as prerelease,
+    semverCompare as compare,
+    semverRcompare as rcompare,
+    semverCompareLoose as compareLoose,
+    semverCompareBuild as compareBuild,
+    semverSort as sort,
+    semverRsort as rsort,
+  };
+  import type semverGt from "semver/functions/gt"
+  import type semverLt from "semver/functions/lt"
+  import type semverEq from "semver/functions/eq"
+  import type semverNeq from "semver/functions/neq"
+  import type semverGte from "semver/functions/gte"
+  import type semverLte from "semver/functions/lte"
+  import type semverCmp from "semver/functions/cmp"
+  import type semverCoerce from "semver/functions/coerce"
+  declare export {
+    semverGt as gt,
+    semverLt as lt,
+    semverEq as eq,
+    semverNeq as neq,
+    semverGte as gte,
+    semverLte as lte,
+    semverCmp as cmp,
+    semverCoerce as coerce,
+  };
+  import type semverSatisfies from "semver/functions/satisfies"
+  import type semverMaxSatisfying from "semver/ranges/max-satisfying"
+  import type semverMinSatisfying from "semver/ranges/min-satisfying"
+  import type semverToComparators from "semver/ranges/to-comparators"
+  import type semverMinVersion from "semver/ranges/min-version"
+  import type semverValidRange from "semver/ranges/valid"
+  import type semverOutside from "semver/ranges/outside"
+  import type semverGtr from "semver/ranges/gtr"
+  import type semverLtr from "semver/ranges/ltr"
+  import type semverIntersects from "semver/ranges/intersects"
+  import type simplify from "semver/ranges/simplify"
+  import type rangeSubset from "semver/ranges/subset"
+  declare export {
+    semverSatisfies as satisfies,
+    semverMaxSatisfying as maxSatisfying,
+    semverMinSatisfying as minSatisfying,
+    semverToComparators as toComparators,
+    semverMinVersion as minVersion,
+    semverValidRange as validRange,
+    semverOutside as outside,
+    semverGtr as gtr,
+    semverLtr as ltr,
+    semverIntersects as intersects,
+    simplify as simplifyRange,
+    rangeSubset as subset,
+  };
+  import type SemVer from "semver/classes/semver"
+  import type Range from "semver/classes/range"
+  import type Comparator from "semver/classes/comparator"
+  declare export { SemVer, Range, Comparator };
```

Comment:
That does not work either, your mixing normal exports and types.

Reply:
The main issue here is that you shouldn't be importing `semver/classes/semver` as a type, `import typeof SemVer from 'semver/classes/semver';` should be enough based on the type def of `semver/classes/semver`